src/science/soilsnow/cbl_soilsnow_main.F90

```diff
@@ -122,7 +122,7 @@ SUBROUTINE soil_snow(dels, soil, ssnow, canopy, met, bal, veg)
     ! snow aging etc...
     CALL snowl_adjust(dels, ssnow, canopy )
 
-   CALL stempv(dels, canopy, ssnow, soil, soil%heat_cap_lower_limit )
+   CALL stempv(dels, canopy, ssnow, soil, REAL(soil%heat_cap_lower_limit) )
```

Comment:
The only way I can see that the `REAL()` is needed is if there a change in precision on `soil%heat_cap_lower_limit` across the SUBROUTINE call. That suggests that we need to actually make the edit in `stempv` not here - or the `REAL()` isn't needed.

Ditto for the call to `soilfreeze` below.